packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuTop/Balance/template.success.js

```diff
@@ -85,6 +90,14 @@ const Success = props => {
     }
   }
 
+  const getSubBalances = () => {
+    return btcUnspendableContext.length || bchUnspendableContext.length ? <SubItems>
+      <Separator margin='0' />
```

Comment:
does this work? should be `style={{margin: 0}}` right?

Reply:
margin gets passed to Separator